### PR TITLE
Implement conferenc created and ended event + invocation on RN side

### PR DESCRIPTION
**Pull request by kbetl-dlb**

### CODE REVIEW

File: android/src/main/java/io/dolby/sdk/comms/reactnative/services/RNNotificationServiceModule.kt [hunk 1]
```diff
@@ -106,19 +106,24 @@ class RNNotificationServiceModule constructor(
    * @param subscribeRNList An array of the subscribed subscription types.
    */
   @ReactMethod
-  fun subscribe(subscribeRNList: ReadableArray) {
+  fun subscribe(subscribeRNList: ReadableArray, promise: ReactPromise) {
```
Comment (graduad) [inline android/src/main/java/io/dolby/sdk/comms/reactnative/services/RNNotificationServiceModule.kt:109]: Has the RN code been changed accordingly? And why is this necessary? How did it work before?

File: src/services/notification/__tests__/index.test.ts [hunk 2]
```diff
@@ -67,13 +67,35 @@ describe('NotificationService', () => {
     });
   });
 
+<<<<<<< HEAD
```
Comment (mateuszglapiak-dolby) [inline src/services/notification/__tests__/index.test.ts:70]: did you push conflict here?

File: src/services/notification/events.ts [hunk 3]
```diff
@@ -29,7 +31,26 @@ export interface ConferenceStatusEventType {
   /** The list of the conference participants. */
   participants: [Participant];
 }
+
+/** The NotificationConferenceCreatedEventType interface gathers information about the received invitation. */
+export interface NotificationConferenceCreatedEventType {
```
Comment (mateuszglapiak-dolby) [inline src/services/notification/events.ts:36]: Do we need `Notification` prefix here?

File: android/src/main/java/io/dolby/sdk/comms/reactnative/services/RNNotificationServiceModule.kt (hunk 1 above)
Comment (kbetl-dlb) [reply android/src/main/java/io/dolby/sdk/comms/reactnative/services/RNNotificationServiceModule.kt:109]: no it's not change. In react for android last parameter is a return type in js/ts code. I forget send promise back for subscription form android to RN

File: src/services/notification/__tests__/index.test.ts (hunk 2 above)
Comment (kbetl-dlb) [reply src/services/notification/__tests__/index.test.ts]: I forget to resolve conflict in one file but and fix it now

**Review by graduad — APPROVED** [review]

**Review by mateuszglapiak-dolby — APPROVED** [review]

**Review by mateuszglapiak-dolby — APPROVED** [review]
